tests/hacker.test.js

```diff
@@ -15,6 +15,25 @@ const storedHacker1 = util.hacker.HackerA;
 const newHacker1 = util.hacker.newHacker1;
 const invalidHacker1 = util.hacker.invalidHacker1;
 
+describe("GET hacker", function () {
```

Comment:
For the future, look into testing failure cases too

Reply:
Deferred